components/Feed/index.js

```diff
@@ -62,6 +66,28 @@ const greetingSubscription = gql`
 `
 
 class Feed extends Component {
+  constructor (props, ...args) {
+    super(props, ...args)
+
+    this.state = {
+      showSearch: false
+    }
+
+    this.onSearchClick = () => {
+      const showSearch = this.state.showSearch
+      this.setState({
+        showSearch: !showSearch
+      })
+      if (showSearch) {
```

Comment:
you should always update the url. now it is inconsistent. If one clicks the icon anywhere but the feed they land on `feed?search=` if they click it while on the feed the url does not change. Correct would be `feed?search` (should work with the value `true`) always and `feed` when dismissing.

```diff
-if (showSearch) {
+Router.replaceRoute(
+  'feed',
+  showSearch? {search: true} : {},
+  { shallow: true }
+)
-}
```